code/modules/client/preferences.dm

```diff
@@ -1255,6 +1255,11 @@ NOTE:  The change will take effect AFTER any current recruiting periods."}
 						if(skin_c)
 							s_tone = max(min(round(skin_c),6),1)
 							to_chat(user,"You will now be [skintone2racedescription(s_tone,species)] in color.")
+					else if(species == "Grey")
+						var/skin_c = input(user, "Choose your Greys's skin color:\n(1 = Gray, 2 = Light, 3 = Green, 4 = Blue)", "Character Preference") as num|null
```

Comment:
maybe null is the reason why 0 shows up?

Reply:
I think that says that it has to be a number entered, or else it enters nothing. The vox color chooser also uses it, so that's probably not it.